Sources/StreamChatUI/CommonViews/Attachments/AttachmentCollectionViewCell.swift

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public protocol AttachmentView: _View {
+    var content: AttachmentPreview? { get set }
+    var discardButtonHandler: (() -> Void)? { get set }
+}
```

Comment:
nit: `AttachmentPreviewView`? Rather than from `_View` it would be better if it inherited from `UIView` to make it possible to use custom attachments